src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -126,6 +127,9 @@ public sealed unsafe partial class Database : IDisposable
         private static readonly C4DocumentObserverCallback _DocumentObserverCallback = DocObserverCallback;
         private static readonly C4DatabaseObserverCallback _DatabaseObserverCallback = DbObserverCallback;
 
+        public const string DefaultScopeName = "_default";
+        public const string DefaultCollectionName = "_default";
```

Comment:
Don't use public const, it is bad for API since the value is copied into the consuming assembly (i.e. not referencing it at runtime).  It means that any change would be an API break.  If it is public, use `static readonly`

Reply:
If the default value is static readonly and I use the default value as public IReadOnlyList<ICollection> GetCollections(string scope = DefaultScopeName), I will be getting _Default parameter for value must be a compile time constant_..

Reply:
If you want to use a constant, you can have a private const, then have the public static readonly be the value of that const.